examples/basics/src/layouts/Layout.astro

```diff
@@ -11,3 +11,32 @@
 		<slot />
 	</body>
 </html>
+
+<style is:global>
```

Comment:
Hmm, it'd be great if the vast majority of the styling was in the Welcome component, is that possible?

Reply:
Sure, it's already global, so it should just work.